message_test.go

```diff
@@ -60,7 +60,7 @@ var (
 		0xFF, 0xFF, 0xFF, 0xFF, // key
 		0x00, 0x00, 0x00, 0x0d, // len
 		// ZSTD data
-		0x28, 0xb5, 0x2f, 0xfd, 0x24, 0x00, 0x01, 0x00, 0x00, 0x99, 0xe9, 0xd8, 0x51,
```

Comment:
4 missing bytes are an optional checksum. `zstd` cli adds it, but DataDog's package does not and I don't see an option to enable it.

There's a message level checksum and checksum, so I think it's fine to leave it out.

Reply:
Opened an issue, but it should not be a blocker here:

* https://github.com/DataDog/zstd/issues/43